README.md

```diff
@@ -51,6 +51,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Ayat Digital Pvt. Ltd.](https://www.ayatdigital.com/) | Balkhu, Kathmandu | We innovate, discover, invent and integrate technology to simplify and ease lives
 - [Axios Softwork](https://axiossoftworks.com/) | Banepa, Kavrepalanchowk | Offer commercial organizations cutting-edge intelligent technical solutions and services on web development and mobile app development
 - [Bajra Technologies](https://bajratechnologies.com/) | Tinkune, Kathmandu | We are a technology company that primarily focuses and excels in building software which bring our customer's ideas to life
+- [BatchNepal Consultancy](https://bajratechnologies.com/) | Bishalnagar, Dhangadhi | We have unbroken streak of happy customers since 2023 and we are still counting. From startup to acquisitions we handle it for you.
```

Comment:
@ChetSocio Thanks for contributing.

The description is not clear enough. Can we update it, it should mention what the company does.

Reply:
Yup! Update as:
 [BatchNepal Consultancy](https://batchnepal.com) |  BatchNepal, a Nepal-based leader in IT services, consulting, outsourcing, and business solutions, leverages technology for business transformation and success for all.

Reply:
@ChetSocio Thanks for contributing. I have updated the URL and the description of the company.